deployment/helm/resource-management-policies/topology-aware/values.yaml

```diff
@@ -0,0 +1,25 @@
+# Default values for nri-plugins.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+---
+image:
+  name: ghcr.io/containers/nri-plugins/nri-resource-policy-topology-aware
+  # tag, if defined will use the given image tag, otherwise Chart.AppVersion will be used
+  tag: unstable
+  pullPolicy: Always
+
+namespace: kube-system
+
+config:
+  Active: topology-aware
```

Comment:
Is this setting really needed? @klihub @jukkar ?

Reply:
I have seen us using it in some places and not using in other. Probably can work without that...
But Krisztian and Jukka are the best to confirm.

Reply:
This just doesn't make any sense and is very confusing to the user as the image should determine what policy to use.